XenModel/XenAPI/VM.cs

```diff
@@ -3798,6 +3798,20 @@ public virtual vm_power_state power_state
         private vm_power_state _power_state;
 
         /// <summary>
+        /// Call a XenAPI plugin on this vm

```

Comment:
You can use the one which was built on trunk, or on the trunk-ring3 branch if it's not yet on trunk. We will update it closer to release time anyway.